docker-compose.yml

```diff
@@ -58,20 +56,15 @@ services:
     networks:
 # Temporary WA - Force container to only have a single interface so mDNS advertisements test correctly - See: https://github.com/AMWA-TV/nmos-testing/issues/214
 #     private-nmos:
-#       ipv4_address: '172.30.101.5'
       external:
-        ipv4_address: '192.168.6.103'
+        ipv4_address: '192.168.1.103'
```

Comment:
Typo, @pkeroulas ??

Reply:
My intention is to remove the private network entirely... Hopefully you'll see a commit later today